src/app/modules/users/components/users-list/users-list.component.html

```diff
@@ -1,11 +1,6 @@
 <div class="table-responsive">
-  <table
-    *ngIf="users"
-    class="table table-sm table-bordered table-striped mb-0"
-    datatable
-    [dtTrigger]="dtTrigger"
-    [dtOptions]="dtOptions"
-  >
+  <table *ngIf="users" class="table table-sm table-bordered table-striped mb-0" datatable [dtTrigger]="dtTrigger"
```

Comment:
@jcalarcon98 restore to original format or previous indentation code

Reply:
Solved!